package.json

```diff
@@ -19,12 +20,16 @@
     ]
   },
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=12.0.0"
   },
   "private": true,
+  "resolutions": {
+    "@heroku-cli/command/**/@oclif/core": "1.26.2"
```

Comment:
This is questionable. Our options are to update @heroku-cli/command to this @oclif/core version or ideally list it as a peerDependency there instead.

Reply:
Thankfully its an easy upgrade. I went ahead and submitted a [PR](https://github.com/heroku/heroku-cli-command/pull/97).

Reply:
Updated to the new version. Thanks.